src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -278,6 +291,31 @@ export default {
       return publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
     },
 
+    viewURI(asset_id, name) {
+      const file_ext = this.fileExtension(name);
+      if (file_ext === 'nii') {
+        return `https://bioimagesuiteweb.github.io/alphaapp/viewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
```

Comment:
Can now be
```suggestion
        return `https://bioimagesuiteweb.github.io/unstableapp/viewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
```
I believe for a bit more stable version